Formula/koka.rb

```diff
@@ -0,0 +1,53 @@
+class Koka < Formula
+  desc "Compiler for the Koka language"
+  homepage "http://koka-lang.org"
+  url "https://github.com/koka-lang/koka/releases/download/v2.3.8/koka-v2.3.9-source.tar.gz"
+  sha256 "ae3e8127b46e51cc4df3525c5f913ecfbaeeb29f0a268e4a3d2e0ac809bc987f"
+  license "Apache-2.0"
+  head "https://github.com/koka-lang/koka.git", branch: "dev"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+  depends_on "vcpkg"
+
+  def install
+    system "cabal", "new-update"
+    system "cabal", "new-build", "-j"
+    system "cabal", "new-run", "koka", "--",
+           "-e", "util/bundle.kk", "--",
+           "--prefix=bundle/local-brew", "--postfix=brew"
+    share.install Dir["bundle/local-brew/share/*"]
+    lib.install Dir["bundle/local-brew/lib/*"]
+    bin.install Dir["bundle/local-brew/bin/*"]
```

Comment:
Why not
```suggestion
           "--prefix=#{prefix}", "--postfix=brew"
```
?

Reply:
Thank you for the super fast reply :-) The reason is that the file times in share should be less than the ones in lib as it contains (koka) pre-compiled modules. If this is not the case, the koka compiler dependency analysis gets confused. 

To ensure this, I decided to explicitly copy the `share` files first, followed by the `lib` files.

Reply:
Also: the `--prefix=bundle/local-brew` flag is not related at all to the real install prefix but used by the koka bundler to create a correct local install tree from which we eventually install. It will not work correctly if we try to bundle directly into the real install prefix.  (I will rename that flag in the future to avoid the confusion)

Reply:
I gave this more thought, and it turns out we can indeed bundle/install directly into the prefix; commit has been pushed.  I think the final formula looks quite good now -- much shorter :-)